pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -133,6 +133,67 @@ func (info *reconcileInfo) publishEvent(reason, message string) {
 	info.events = append(info.events, info.host.NewEvent(reason, message))
 }
 
+// hostConfigData imaplementation of cons configuration data interface
```

Comment:
```suggestion
// hostConfigData implementation of cons configuration data interface
```

I'm not sure what "cons" means in that sentence. It would be good to refer to the provisioner.HostConfigData interface by name.

Reply:
Just typo :)